### DISCUSSION TITLE
Quick Charge

### DISCUSSION THREAD

**Discussion by jonny190** [celsworth/lxp-bridge#258] [General]
Is there a way to enable Quick Charge via Home Assistant? Currently on Octopus with an Ohme car charger they give sporadic cheap rates for charging my car and I'd like to charge my battery at these times as well

However it seems that i just turn on "AC Charge" and/or "Charge Priority" this does not have the same effect as pressing the "Start Quick Charge" button in the web interface or app

**Comment by jacovt** [comment]
Hello @jonny190 ,

I believe this has been solved in #180.
